features/dev/verblijfplaats-binnenland-verblijfadres.feature

```diff
@@ -21,10 +21,10 @@ Functionaliteit: Persoon: verblijfplaats binnenland - verblijfadres
     | type | Adres  |
 
     Voorbeelden:
-    | gba naam           | gba waarde | naam                     |
-    | straatnaam (11.10) | .          | verblijfadres.straat     |
-    | huisnummer (11.20) | 0          | verblijfadres.huisnummer |
-    | woonplaats (11.70) | .          | verblijfadres.woonplaats |
+    | gba naam           | gba waarde | naam                              |
+    | straatnaam (11.10) | .          | verblijfadres.officieleStraatnaam |
+    | huisnummer (11.20) | 0          | verblijfadres.huisnummer          |
+    | woonplaats (11.70) | .          | verblijfadres.woonplaats          |
 
   Abstract Scenario: verblijfadres veld(en) van persoon is/zijn in onderzoek
```

Comment:
Deze scenario kan weg. Het staat ook in features/bevragen/persoon/verblijfplaats/adres/verblijfadres/in-onderzoek.feature

Reply:
Als je het verwijderd kijk dan of mijn onderstaande opmerking daar (features/bevragen/persoon/verblijfplaats/adres/verblijfadres/in-onderzoek.feature) ook van toepassing is.